src/axolotl/train.py

```diff
@@ -259,11 +259,27 @@ def terminate_handler(_, __, model_weakref):
         model.save_pretrained(cfg.output_dir, safe_serialization=safe_serialization)
 
     if not cfg.hub_model_id:
+        from huggingface_hub import HfApi
+        from huggingface_hub.utils import RepositoryNotFoundError
+
         try:
-            trainer.create_model_card(
-                model_name=cfg.output_dir.lstrip("./").encode("utf-8").decode("utf-8")
-            )
-        except (AttributeError, UnicodeDecodeError):
+            # Check to make sure the base model is from HuggingFace not a local directory
+            hf_api = HfApi()
+            hf_api.model_info(cfg.base_model)
+
+            model_card_kwarg = {"model_name": cfg.output_dir.lstrip("./")}
```

Comment:
Saw this part removed `.encode("utf-8").decode("utf-8")` from model_name. Is this intentional? I'm not sure what it's for though.

Reply:
oh you are right, I did unintentionally remove that part, will restore them. Thanks for picking it up!